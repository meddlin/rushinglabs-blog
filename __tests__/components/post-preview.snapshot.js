import { render } from '@testing-library/react';
import PostPreview from '../../components/post-preview';

it('renders PostPreview unchanged', () => {
    const { container } = render(<PostPreview />);

    expect(container).toMatchSnapshot()
})