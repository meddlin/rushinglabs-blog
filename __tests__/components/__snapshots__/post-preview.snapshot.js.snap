// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PostPreview unchanged 1`] = `
<div>
  <li
    class="mb-10"
  >
    <div
      class="flex flex-col"
    >
      <a
        class="grow text-base text-black"
        href="/blog/undefined/undefined"
      />
    </div>
    <p
      class="text-sm text-slate-700"
    />
  </li>
</div>
`;
